helix-core/src/main/java/org/apache/helix/BaseDataAccessor.java

```diff
@@ -43,6 +43,18 @@
    */
   boolean create(String path, T record, int options);
 
+  /**
+   * This will always attempt to create the znode, if it exists it will return false. Will
+   * create parents if they do not exist. For performance reasons, it may try to create
+   * child first and only if it fails it will try to create parents
+   * @param path path to the ZNode to create
+   * @param record the data to write to the ZNode
+   * @param options Set the type of ZNode see the valid values in {@link AccessOption}
+   * @param ttl TTL of the node in milliseconds, if options supports it
+   * @return true if creation succeeded, false otherwise (e.g. if the ZNode exists)
+   */
+  boolean create(String path, T record, int options, long ttl);
```

Comment:
So this only applies to create? It means this property cannot be added in the middle by override or set?